gradle.properties

```diff
@@ -45,3 +45,7 @@ POM_SCM_DEV_CONNECTION=git@github.com:linkedin/Spyglass.git
 POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
+
+#For AndroidX migration
+android.useAndroidX=true
+android.enableJetifier=true
```

Comment:
Have we tested using this flag will make glide compatible with AndroidX? There is an issue here: https://github.com/bumptech/glide/issues/3080

Reply:
Hi @csv8674xn , yes we have tested this flag. Using this flag will make Glide compatible with AndroidX. We didn't encounter any compile time or runtime issues.